pkgs/tools/misc/blivet-gui/default.nix

```diff
@@ -0,0 +1,62 @@
+# Notes for using this package outside of NixOS:
+# 1. --pure cannot be used (as pkexec will be used from the path,
+#    and we can't use nixpkgs polkit due to lack of setuid bit)
+# 2. You must prefix the blivet-gui command with "SHELL=/bin/bash"
+#    (otherwise your system polkit will reject the SHEL Lfrom nixpkgs).
```

Comment:
RFC, is there a better way?

Reply:
You should not execute scripts as root in the first place, the program should have a small privileged utility that can be run from unprivileged GUI app.

Reply:
Er, that's exactly what it does (`blivet-gui` runs `blivet-gui-daemon` via `pkexec`). The problem is that `pkexec` looks at `SHELL` and refuses to run anything if it's not in `/etc/shells`. This is going to be the case on non-NixOS.